client/src/components/blocks/TrackSetting/index.tsx

```diff
@@ -49,6 +44,16 @@ function TrackSetting({ projectTitle, bpmState, trackTags, tracks }: ProjectInfo
     },
   });
 
+  const handleTrackSubmit = () => {
+    const recordedAudio = audioList.find(({ id }) => id === NEW_TRACK_ID);
+
+    if (!title || !trackTag || !recordedAudio) {
+      return;
+    }
```

Comment:
제목, 태그, 녹음에 대한 사용자 액션을 유도하는 얼럿을 추가해주는 것은 어떨까요??
```suggestion
    const requiredProperties = [title, trackTag, recordedAudio];
    const errorPropertyIdx = requiredProperties.fineIndex((item) => !item);
  
    if (errorPropertyIdx > -1) {
      alert(`${requiredProperties[errorPropertyIdx]}을 채워주세요.`);
      return;
    }
```

Reply:
오.. 이렇게 required 요소를 체크하는 방법 좋은 것 같아요! 👍🏻